src/themes/hel/index.js

```diff
@@ -1,24 +1,26 @@
 let Colors = require('material-ui/colors')
 let Spacing = require('material-ui/styles/spacing')
 import {createMuiTheme} from 'material-ui/styles'
-import blue from 'material-ui/colors/blue'
+import green from 'material-ui/colors/green'
```

Comment:
$hel-green or something?

Reply:
These are colors from Material-Ui, they do not have the Helsinki variables

Reply:
Also, No way to implement the Helsinki color variables and other variables since the local scss is broken .